tests/steps/other_v2_steps.py

```diff
@@ -97,6 +98,31 @@ def read_program(context, path):
     return read_program_binary(path)
 
 
+# Send transactions to progress block numbers on dev mode.
+def send_zero_transactions(context, txns=1):
+    sp = context.app_acl.suggested_params()
+    for _ in range(txns):
+        payment = transaction.PaymentTxn(
+            context.accounts[0],
+            sp,
+            constants.ZERO_ADDRESS,
+            random.randint(100000, 900000),
+        )
+        signed_payment = context.wallet.sign_transaction(payment)
+        context.app_acl.send_transaction(signed_payment)
+        # Wait and confirm that the zero payment succeeded.
+        transaction.wait_for_confirmation(
```

Comment:
Is the number of rounds that confirmations are waited for predictable in _dev mode_?

Reply:
It's more or less always 1 (the block is finalized after the transaction is sent). 3 and 5 are mostly magic numbers right now - I could consolidate them into one for cleanliness.

Reply:
I don't feel strongly. Knowing that an exact amount is deducted each time gives a way of calculating how many such transactions were called (but perhaps in dev mode, the round number gives similar info)

Reply:
I think this would be a good place to consolidate as you suggest. Also, hopefully we would discover at this juncture that 1 doesn't work. We could have `wait_for_confirmation()` default to 1 round and not even pass that parameter into it.

Reply:
Same thought applies to the new `dev_mode_wait_for_confirmation()`